docs/source/guide/distributed-hetero.rst

```diff
@@ -0,0 +1,94 @@
+.. _guide-distributed-hetero:
+
+7.3 Distributed Heterogeneous graph training
+--------------------------------------------
+
+DGL v0.6.0 provides an experimental support for distributed training on heterogeneous graphs.
+In this version, each node or edge is assigned with two IDs: homogeneous ID and type-wise ID.
```

Comment:
The Line 7th does not give too much useful information, and the below sentences clearly explain what is different. So suggest to remove it or revise it to spotlight this is the key change.